packages/expo-errors/src/__tests__/Errors-test.ts

```diff
@@ -1,10 +1,98 @@
 import * as Errors from '../Errors';
 
 describe('UnavailabilityError', () => {
-  it('has a constructor which takes a module and property name as parameters', () => {
+  it(`has a constructor which takes a module and property name as parameters`, () => {
     let err = new Errors.UnavailabilityError('TestModule', 'someProperty');
     expect(err.code).toBe('ERR_UNAVAILABLE');
     expect(err.message).toContain('TestModule');
     expect(err.message).toContain('someProperty');
   });
 });
+
+describe('deprecate', () => {
```

Comment:
Prefer moving these tests into `__tests__/deprecate-test.ts` -> this way we can run more of the tests in parallel